ldms/src/core/ldms_xprt.c

```diff
@@ -2469,10 +2470,9 @@ static int __ldms_conn_msg_verify(struct ldms_xprt *x, const void *data,
  */
 static void ldms_zap_cb(zap_ep_t zep, zap_event_t ev)
 {
-	struct ldms_xprt_event event = {
-			.type = LDMS_XPRT_EVENT_LAST,
-			.data = NULL,
-			.data_len = 0};
+	struct ldms_xprt_event event;
```

Comment:
How is this different than the static initialization?

Reply:
holes in the structure { (int, ptr, int) doesn't align nicely } get init with memset but not the initializer. 
An alternate fix would be put .data last (int, int, ptr),  but I'm not sure if this goes over the wire or not.

Reply:
I see. OK. Thanks for the explanation.